src/Service/AbstractService.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/).
+ *
+ * @link      http://github.com/zendframework/zend-version for the canonical source repository
+ * @copyright Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+namespace Zend\Version\Service;
+
+use Zend\Version\Exception\InvalidFormatException;
+use Zend\Version\Version;
+
+/**
+ * A generic implementation of the ServiceInterface.
+ */
+abstract class AbstractService implements ServiceInterface
```

Comment:
Actually, nvm, it is used, but I'd still rather duplicate the codethan having an abstract type for these few lines of code